### DISCUSSION TITLE
me when:

### DISCUSSION THREAD

**Discussion by ThatOneSillyGuyDon** [mayo78/vsRobloxButNotReally#11] [Made something!]
![image](https://user-images.githubusercontent.com/87618246/161405356-517320fd-fd26-4356-b039-0cc339018d5a.png)

**Comment by ThatOneSillyGuyDon** [comment]
Pixel Splash, Screen Resolution & Fullscreen Options Code by HiroMizuki

Custom Player Note Skin Support Code by Phoneguytech75

**Reply by Theoyeah** [reply]
YO YOU ADDED THE NOTZSKIN OPTION LIKE ME
